x-pack/plugins/apm/public/components/app/trace_explorer/trace_search_box/index.tsx

```diff
@@ -23,12 +22,11 @@ import {
 } from '../../../../../common/trace_explorer';
 import { useApmDataView } from '../../../../hooks/use_apm_data_view';
 import { useApmPluginContext } from '../../../../context/apm_plugin/use_apm_plugin_context';
-import { EQLCodeEditorSuggestionType } from '../../../shared/eql_code_editor/constants';
```

Comment:
Don't we need to get rid of these files as well?

Reply:
I discussed this with @sqren . I will remove the existing component in a follow up PR.
did not wanted to pollute this PR with too many changes